app/code/Magento/Wishlist/Block/Customer/Wishlist.php

```diff
@@ -29,6 +29,11 @@ class Wishlist extends \Magento\Wishlist\Block\AbstractBlock
      */
     protected $_helperPool;
 
+    /**
+     * @var  \Magento\Wishlist\Model\ResourceModel\Item\Collection
+     */
+    protected $_collection;
```

Comment:
Please change to private

Reply:
Access level to Magento\Wishlist\Block\Customer\Wishlist::$_collection must be protected (as in class Magento\Wishlist\Block\AbstractBlock) or weaker in /home/users/ratnesh/www/html/magento/m2.3-1/app/code/Magento/Wishlist/Block/Customer/Wishlist.php on line 276

not suggested to do so.